pkg/kubectl/cmd/get.go

```diff
@@ -501,7 +513,9 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 			}
 			continue
 		}
-		if err := printer.PrintObj(decodedObj, w); err != nil {
+		// TODO: understand if passing original is safe here
+		// if err := printer.PrintObj(decodedObj, w); err != nil {
+		if err := printer.PrintObj(original, w); err != nil {
```

Comment:
I think we should use `decodedObj`, original may be an unversioned object.
Is there any issue when using `decodedObj`?